### PR TITLE
(SLV-555) Add Ace and Bolt to processes tracked

### PR DESCRIPTION
**Pull request by RandellP**
expanded the expression that finds process to track to include ace server and bolt server as puma servers.
updated the code that converts the command text into a simpler string that represents the command
updated the baseline query to grab the newly added fields for ace and bolt

### CODE REVIEW

File: tests/helpers/perf_run_helper.rb [hunk 1]
```diff
@@ -917,8 +917,14 @@ def push_to_bigquery
   def get_process_hash(perf_result_processes)
     process_hash = {}
     perf_result_processes.keys.each do |key|
-      # All of the puppet processes we care about are jars
-      process_match = perf_result_processes[key][:cmd].match(%r{.*/([a-z,\-]*)\.jar})
+      # Most of the puppet processes we care about are jars
+      # Two are puma servers
+      full_cmd = perf_result_processes[key][:cmd]
+      process_match = if full_cmd.match(/puma /)
```
[the remark this answers is not in the document]
Comment (RandellP) [reply tests/helpers/perf_run_helper.rb]: it was originally, but rubocop changed it to this.

File: tests/helpers/perf_run_helper.rb [hunk 2]
```diff
@@ -819,7 +819,7 @@ def copy_archive_files
   end
 
   def get_perf_result
-    perf = stop_monitoring(master, "/opt/puppetlabs")
+    perf = stop_monitoring(master, "-E '/opt/puppetlabs|puma.*/etc/puppetlabs'")
```
[the remark this answers is not in the document]
Comment (RandellP) [reply tests/helpers/perf_run_helper.rb]: SLV-586 filed

File: tests/helpers/perf_run_helper.rb [hunk 3]
```diff
@@ -917,8 +917,14 @@ def push_to_bigquery
   def get_process_hash(perf_result_processes)
     process_hash = {}
     perf_result_processes.keys.each do |key|
-      # All of the puppet processes we care about are jars
-      process_match = perf_result_processes[key][:cmd].match(%r{.*/([a-z,\-]*)\.jar})
+      # Most of the puppet processes we care about are jars
+      # Two are puma servers
```
[the remark this answers is not in the document]
Comment (RandellP) [reply tests/helpers/perf_run_helper.rb]: done

**Comment by RandellP** [comment]
@johnduarte processed all of you feedback, ready for another look.

**Comment by johnduarte** [comment]
> @johnduarte processed all of you feedback, ready for another look.

Waiting for a tech-debt ticket to be opened for the `egrep` issue.

File: tests/helpers/perf_run_helper.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (billclaytor) [reply tests/helpers/perf_run_helper.rb]: Please add a code comment briefly explaining the hack and referencing SLV-586 :)

[the remark this answers is not in the document]
Comment (RandellP) [reply tests/helpers/perf_run_helper.rb]: done

**Pull request merged**